README.md

```diff
@@ -95,8 +95,9 @@ Can be a [glob](https://github.com/isaacs/node-glob#glob-primer) or and `Array`
 | -------- | ----------------------------------------------------------------------------------------------------------- | ------------------------------------ |
 | `path`   | **Required.** A [glob](https://github.com/isaacs/node-glob#glob-primer) to identify the files to upload.    | -                                    |
 | `label`  | Short description of the file displayed on the GitLab release. Ignored if `path` matches more than one file.| File name extracted from the `path`. |
-| `type` | Asset type displayed on the GitLab release. Can be `runbook`, `package`, `image` and `other` (see official documents on [release assets](https://docs.gitlab.com/ee/user/project/releases/#release-assets)). | `other` |
+| `type` | Asset type displayed on the GitLab release. Can be `runbook`, `package`, `image` and `other` (see official documents on [release assets](https://docs.gitlab.com/ee/user/project/releases/#release-assets)). If this is set to `generic_package` it will upload the `glob` as a [generic package](https://docs.gitlab.com/ee/user/packages/generic_packages/). | `other` |
```

Comment:
I like the simplicity of reusing the existing option! I don't like overloading the semantics of this option. I think I'd prefer a separate option (maybe `storage` with possible values `project_upload` and `generic_package`). What's your opinion @JonasSchubert?

Reply:
Adding a new option would keep things in a clean way, indeed.
`type` values would only contain the documented [link types](https://docs.gitlab.com/ee/user/project/releases/#link-types).

Reply:
Instead of `storage` maybe `target`. Also considering the URL feature?

Reply:
@JonasSchubert Yes, I like `target`!